starter-code/scripts/views/articleView.js

```diff
@@ -13,6 +13,7 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  // It populates the filters in the handlebars section by creating unique lists of author names and categories. It's execution path is from aticlaeView.index.
```

Comment:
"It populates the filters in the handlebars section... "
Is there a handlebars section?